packages/credential-provider-ini/src/fromIni.integ.spec.ts

```diff
@@ -158,7 +158,7 @@ describe("fromIni region search order", () => {
     delete process.env.AWS_PROFILE;
   });
 
-  it("should use 1st priority for the clientConfig given to the provider factory", async () => {
+  it.only("should use 1st priority for the clientConfig given to the provider factory", async () => {
```

Comment:
Probably introduced during debugging
```suggestion
  it("should use 1st priority for the clientConfig given to the provider factory", async () => {
```